crates/bevy_reflect/src/path/mod.rs

```diff
@@ -271,22 +275,32 @@ pub trait GetPath: Reflect {
 // Implement `GetPath` for `dyn Reflect`
 impl<T: Reflect + ?Sized> GetPath for T {}
 
+/// An [`Access`] combined with an `offset` for extra context.
```

Comment:
These docs aren't clear to me: what does `offset` mean?

Reply:
Fixed.